srcpkgs/povray/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'povray'
+pkgname=povray
+version=3.7.0.7
+revision=1
+build_style=gnu-configure
+_v=${version%.*.*}
+conf_files="
+	/etc/povray/$_v/povray.conf
+	/etc/povray/$_v/povray.ini"
+nocross="Builds fail on ARM architectures, while on x86_64-musl it: (1) fails to build or (2) if patched to make it build correctly it segfaults on startup. Logs are at https://travis-ci.org/void-linux/void-packages/builds/412583835."
+configure_args="COMPILED_BY=Void"
+hostmakedepends="git autoconf automake"
+makedepends="boost-devel glu-devel zlib-devel libpng-devel libjpeg-turbo-devel tiff-devel SDL-devel"
```

Comment:
Line length shouldn’t exceed 80 characters. Just break the line after libjpeg-turbo-devel and indent the new line with a single space.

Reply:
Or shorten it to `boost-devel libpng-devel SDL-devel tiff-devel`

Reply:
Have you tried that and found it builds fine? Because I tried without libjpeg-turbo-devel and it gave me configure errors. Same goes with all those make dependencies.

Reply:
Well I'll be, just gave it a go and it worked fine. I suppose that was when it was in `hostmakedepends`...

Reply:
That was determined using xmindep